u-mad-bro/src/Layout/Landing.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from "react";
+import Logo from "../components/Logo.js";
+import { Drawer, Row, Col } from "antd";
+import { UMBButton } from "../theme/Styled.js";
+import "../scss/Landing.scss";
+
+import SignIn from "../SignInLayout/SignIn.js";
+import SignUp from "../SignInLayout/SignUp.js";
+
+const Landing = () => {
+  const [landingState, setLandingState] = useState({
+    drawer: false,
+    view: ""
+  });
+
```

Comment:
Dude!  This drawer component is epic!  The code is so simple but the result is awesome!  I'm gonna be referencing this component for the rest of my web dev career!